### PR TITLE
feat(core): Add docket link in the template for minute entries

### PR DESCRIPTION
**Pull request by ERosendo**
This PR adds the docket link to the `MINUTE_TEMPLATE`.

### CODE REVIEW

File: bc/core/utils/messages.py [hunk 1]
```diff
@@ -68,6 +68,7 @@ def _available_space(self, *args, **kwargs) -> int:
     def format(self, *args, **kwargs) -> str:
         if "description" in kwargs:
             available_space = self._available_space("description", **kwargs)
+            print(available_space)
```
Comment (mlissner) [inline bc/core/utils/messages.py:71]: Oops.

Comment (mlissner) [reply bc/core/utils/messages.py:71]: Can you fix with an amended commit and a force-push so our commit log isn't too polluted?

Comment (ERosendo) [reply bc/core/utils/messages.py:71]: Sure!

**Comment by mlissner** [comment]
Nice to see that we didn't have to count any characters to make this tweak. :)

**Pull request merged**